clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -577,13 +577,8 @@ describe("hubConnection", () => {
                     const hubConnection = getConnectionBuilder(transportType).build();
                     hubConnection.serverTimeoutInMilliseconds = 100;
 
-                    const timeout = setTimeout(200, () => {
```

Comment:
This was an attempt to get the test to fail sooner than Jasmine's own timeout, but I think it's cleaner to just let Jasmine do it's thing.